npm/design-system/package.json

```diff
@@ -19,11 +19,11 @@
     "watch": "yarn build --watch"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.2",
-    "@fortawesome/fontawesome-svg-core": "^1.2.34",
-    "@fortawesome/free-brands-svg-icons": "^5.15.2",
-    "@fortawesome/free-solid-svg-icons": "^5.15.2",
-    "@fortawesome/react-fontawesome": "^0.1.14",
```

Comment:
is this a breaking change for the design-system module?

Reply:
No it is not, there is a visual change on one component that is not used (LeftNav.tsx)